src/openrct2/actions/RideSetSettingAction.cpp

```diff
@@ -50,7 +50,12 @@ GameActions::Result RideSetSettingAction::Query() const
     auto ride = GetRide(_rideIndex);
     if (ride == nullptr)
     {
-        LOG_ERROR("Ride not found for rideIndex %u.", _rideIndex.ToUnderlying());
+        LOG_ERROR(
+            "Ride not found for rideIndex %u, trying to apply setting %d.", _rideIndex.ToUnderlying(), EnumValue(_setting));
```

Comment:
Please get rid of this entirely, within Query the game actions are supposed to ask if something would work so this doesn't make much sense, never did.

Reply:
Would be also good to remove those in all the other ones, no need to do this in this PR but generally speaking.